Jellyfin.Api/Controllers/SessionController.cs

```diff
@@ -61,7 +62,8 @@ public class SessionController : BaseJellyfinApiController
     public ActionResult<IEnumerable<SessionInfo>> GetSessions(
         [FromQuery] Guid? controllableByUserId,
         [FromQuery] string? deviceId,
-        [FromQuery] int? activeWithinSeconds)
+        [FromQuery] int? activeWithinSeconds,
+        [FromQuery] bool? nowPlaying)
```

Comment:
I would expect `nowPlaying=false` to only return sessions that are not playing anything but the current implementation doesn't differ between omitting the parameter and setting it to false.

Reply:
I had not thought of that angle, will implement

Reply:
updated the code